eventual-cassandra/src/main/scala/com/evolutiongaming/kafka/journal/eventual/cassandra/PointerStatements.scala

```diff
@@ -211,20 +214,19 @@ object PointerStatements {
 
     implicit class SelectInOps[F[_]](val self: SelectIn[F]) extends AnyVal {
 
-      def orElse(other: SelectIn[F])(implicit F: Monad[F]): SelectIn[F] = {
-        (topic: Topic, partitions: Nel[Partition]) => {
-          for {
-            offsets <- self(topic, partitions)
-            offsets <- if (offsets.isEmpty) other(topic, partitions) else offsets.pure[F]
-          } yield offsets
-        }
+      def both(other: SelectIn[F])(implicit F: Monad[F], P: Parallel[F]): SelectIn[F] = {
```

Comment:
it's not an optimisation, without this - it can return wrong values, it could return `non` empty set of offsets hence prevent from falling back to old `pointer` table. But data for other partition might not be filled in in new `pointer2` table.